apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -3158,7 +3158,7 @@
     "unlockedBalance": "Unbonded",
     "unlockedTooltip": "Unbonded assets can now be moved using the withdraw operation.",
     "networkFees": "Network fees are automatically set by the Polkadot consensus, you won't be able to review them on your device",
-    "bondedBalanceBelowMinimum": "Your bonded balance is below the current minimum of {{minimumBondBalance}}. Your nominations are at risk of being removed.",
+    "bondedBalanceBelowMinimum": "As a nominator, a minimum of {{minimumBondBalance}} is required to submit an intention to nominate",
```

Comment:
Your bonded balance is below the current minimum of {{minimumBondBalance}}. You cannot submit nomination requests.

Thoughts ?